kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -70,39 +70,9 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self.cadvisor_legacy_port = inst.get('cadvisor_port', CADVISOR_DEFAULT_PORT)
```

Comment:
It seems like `self.kube_node_labels` is not used anywhere in this check 🤔

Reply:
looks like it's an left over of https://github.com/DataDog/integrations-core/blob/master/kubernetes/conf.yaml.example#L140-L146
@hkaj should it be implemented or dropped?

Reply:
it can be dropped, this has been moved to main agent settings: https://github.com/DataDog/datadog-agent/blob/0aa3114a92a568b8b84a02919789fbc2d9358477/pkg/config/config.go#L176